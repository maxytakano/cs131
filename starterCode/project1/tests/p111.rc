
function : int foo(int array [10][4] ) {
	return array[9][3];
}

function : int bar(int array[10]) {
	return array[0];
}

function : void main() {
	int a [5];
	int b;
	
	b = bar(a);		//error
}