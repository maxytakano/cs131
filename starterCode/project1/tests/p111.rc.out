Error, "/tmp/cs131_testrunner_ZyTaux/file.rc", line 14: 
  Argument of type int[5] not assignable to value parameter array, of type int[10].
Compile: failure.
